### ISSUE TITLE
Access to AudioPlaybackCapture API

### ISSUE THREAD

**Issue by ask6155** [termux/termux-api#721]
**Feature description**
I run a snapserver on my phone which is available as a termux package.
The snapserver can take input in various ways but some of them have been limited due to it running under termux (For eg. I couldn't take ALSA input)
What I have been doing is running music through mpd and piping it to snapserver (all in termux)
What I'd like to do is instead take system output directly from android using the `AudioPlaybackCapture` API and pipe it into my snapserver. That way I can play whatever I want on my phone and the snapclients will receive it.

basically a script which will ask me permission to record system audio and I can configure it to ouput to fifo or stdout

**Reference implementation**

Yes `AudioPlaybackCapture` is available since android 10.

I have used [Roc Droid](https://github.com/roc-streaming/roc-droid) and it works well if I want to share audio to another device. It uses this API I think (I'm not sure)
